CMakeLists.txt

```diff
@@ -213,6 +204,9 @@ endif()
 # add sources of the wrapper as a "SQLiteCpp" static library
 add_library(SQLiteCpp ${SQLITECPP_SRC} ${SQLITECPP_INC} ${SQLITECPP_DOC} ${SQLITECPP_SCRIPT})
 
+# set minimum version to c++11
+target_compile_features(SQLiteCpp PUBLIC cxx_std_11)
```

Comment:
The cxx_std_11 compiler meta-feature is not available CMake 3.5 (which was the minimal version until now, see line 7 cmake_minimum_required(VERSION 3.5))

It was introduced in 3.8, [CMake 3.8 release notes](https://cmake.org/cmake/help/v3.8/release/3.8.html#c-c)

Could you please also update the minimum version?
Thanks!